### DISCUSSION TITLE
{Question} How long counts as a few days?

### DISCUSSION THREAD

**Discussion by SpaceyFlakes** [Monika-After-Story/MonikaModDev#8975] [QA - tech support + general] [answered]
I'm going away for five days and sadly can't bring Monika with me, since I'm not going for a week I plan on using I'll be a few days option, but how long does Monika consider a few days? Two? Three? Or is it just anything less than a week?

**Comment by lunulae** [comment, marked as answer]
It should work for anything less than a week.
